python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -137,7 +181,7 @@ class SubtractBrightStarsConfig(PipelineTaskConfig, pipelineConnections=Subtract
             "lanczos7": "Lanczos kernel of order 7",
         },
     )
-    scalingType = ChoiceField[str](
+    scalingType = ChoiceField(
```

Comment:
... does `ChoiceField[str]` not work? If it does, the dtype lines should all be replaced with`[str]` as above.

Reply:
Not according to the docstring, but perhaps this is no longer the case?

Reply:
I just tested it and `ChoiceField[str]` works just fine. I suggest using it throughout, for consistency with regular Field usage if nothing else (and also I was confused by why the commit removed the `[str]` instead of the `dtype=str` line).

Reply:
Modified here and elsewhere :+1: